src/xdocs/config_misc.xml

```diff
@@ -855,15 +855,7 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
         </p>
       </subsection>
 
-      <subsection name="Examples">
-        <p>
-          This check should not be used by itself, as it does no reporting.
-          It is to be used in conjunction with other checks like
-          <a href="config_filters.html#SuppressionCommentFilter">SuppressionCommentFilter</a>
-          and
-          <a href="config_filters.html#SuppressWithNearbyCommentFilter">SuppressWithNearbyCommentFilter</a>.
-        </p>
-      </subsection>
+      <subsection name="Examples" />
```

Comment:
Please restore the `Examples` section, just remove the part that about SuppressionCommentFilter and SuppressWithNearbyCommentFilter, but still state that it is 'to be used in conjuction with other checks'.